likelihood/testrun.py

```diff
@@ -29,3 +33,66 @@
 pf = {k: bf[k] for k in p0.keys()}
 print("Final params: ")
 print(pf)
+
+#======================DETERMINE ERRORS ON PARAMETERS========================
+
+# remove cobaya_out directory (just for now!) to make running code easier
+os.system('rm -r cobaya_out')  
+
+h = 0.005  # stepsize
```

Comment:
The stepsize here is the same for everything. If you have a parameter whose natural number is 100 then stepsize of 0.005 would be very different from what it would be is the parameter is 1. 
I suggest you have a set of "typical variations" vector, which is the same size of parameters and give you rule of thumb 1 sigma. The stepsize can then be 0.005*number.